drivers/mtd_sdcard/mtd_sdcard.c

```diff
@@ -47,13 +57,33 @@ static int mtd_sdcard_init(mtd_dev_t *dev)
         dev->write_size       = SD_HC_BLOCK_SIZE;
         return 0;
     }
+#elif IS_USED(MODULE_SDMMC)
```

Comment:
but at that point you might as well just create a new `mtd_sdmmc` module and remove the `mtd_sdcard` module once `sdcard_spi` is ported to the new interface.